frontend-dev/src/components/AllIntegrations/SmartSuite/IntegrationHelpers.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-expressions */
+
+export const addFieldMap = (i, confTmp, setConf) => {
+  const newConf = { ...confTmp }
+  newConf.field_map.splice(i, 0, {})
+  setConf({ ...newConf })
+}
+
+export const delFieldMap = (i, confTmp, setConf) => {
+  const newConf = { ...confTmp }
+  if (newConf.field_map.length > 1) {
+    newConf.field_map.splice(i, 1)
+  }
+
+  setConf({ ...newConf })
+}
+
+export const handleFieldMapping = (event, index, conftTmp, setConf) => {
+  const newConf = { ...conftTmp }
+  newConf.field_map[index][event.target.name] = event.target.value
+
+  if (event.target.value === 'custom') {
+    newConf.field_map[index].customValue = ''
+  }
+  setConf({ ...newConf })
+}
+
+export const handleCustomValue = (event, index, conftTmp, setConf) => {
+  const newConf = { ...conftTmp }
+
+  newConf.field_map[index].customValue = event.target.value
+  setConf({ ...newConf })
+}
+
+export const handleCustomField = (event, index, conftTmp, setConf, fieldValue) => {
+  const newConf = { ...conftTmp }
```

Comment:
Provide an explanation of how to use handleCustomValue and handleCustomField, and specify which file contains these functions.